src/local-notifications.ios.ts

```diff
@@ -344,6 +343,12 @@ export class LocalNotificationsImpl extends LocalNotificationsCommon implements
     });
   }
 
+  addOnMessageClearedCallback(onReceived: (data: ReceivedNotification) => void): Promise<any> {
+    // TODO: To be implemented...
```

Comment:
??

Reply:
Just to have feature parity between both platforms it would be nice to implement this as well for iOS. Anyway, it's ok to have features just on one platform.

Also, I actually need to be notified when the notifications are cleared on Android to unpersist them, but on iOS it works differently, so there's no need for that.